mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -275,6 +321,13 @@ class GBTClassificationModel private[ml](
   @Since("2.0.0")
   lazy val featureImportances: Vector = TreeEnsembleModel.featureImportances(trees, numFeatures)
 
+  private def classProbability(loss: String, rawPrediction: Double): Double = {
```

Comment:
Actually, this would be better served embedded in the loss object. One solution would be to make a few changes to the loss: 
````scala
trait ClassificationLoss extends Loss {
  private[spark] def computeProbability(prediction: Double): Double
}
object LogLoss extends ClassificationLoss
````

Then we could add a class member to the model `private val oldLoss: ClassificationLoss = getOldLossType`, then we can just call `oldLoss.computeProbability(pred)` inside `raw2ProbabilityInPlace`. There might be a better solution too, but really I think it should be part of the loss.